go/gosec/bad_temp_file/bad_tmp.go

```diff
@@ -0,0 +1,14 @@
+package samples
+
+import (
+	"fmt"
+	"io/ioutil"
+)
+
+func main() {
```

Comment:
Can we add a True Negative test case as well?

Reply:
Will do!